screensaver.video/addon.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="screensaver.video" name="VideoScreensaver" version="1.3.0" provider-name="robwebset">
+<addon id="screensaver.video" name="VideoScreensaver" version="1.4.0" provider-name="robwebset">
 	<requires>
 		<import addon="xbmc.python" version="2.14.0"/>
 		<import addon="script.module.simplejson" version="3.3.0"/>
 	</requires>
 	<extension point="xbmc.ui.screensaver" library="default.py"/>
-  	<extension point="xbmc.python.script" library="screensaver.py" />
-	<extension point="xbmc.service" library="service.py" start="login">
-		<provides>executable</provides>
+  	<extension point="xbmc.python.script" library="screensaver.py"/>
+	<extension point="xbmc.service" library="service.py" start="login"/>
+	<extension point="xbmc.python.pluginsource" library="plugin.py">
+		<provides>video</provides>
```

Comment:
One thing to consider for the future would be to separate the different extension points, because not everybody who wants the screensaver functionality wants this to appear in the vieos section, too.
This is no blocker , I´m even guilty of this myself (planning to change that though), but would be nice if you would keep this in mind for the future.